packages/server-core/src/projects/project-permission/project-permission.hooks.ts

```diff
@@ -205,6 +206,41 @@ const makeRandomProjectOwner = async (context: HookContext<ProjectPermissionServ
   if (context.id && context.result) await context.service.makeRandomProjectOwnerIfNone(result[0].projectId)
 }
 
+/**
+ * Filter permissions for logged in user
+ * @param context
+ * @returns
+ */
+const filterUserPermissions = async (context: HookContext<ProjectPermissionService>) => {
```

Comment:
Do we need this hook? Can we use `@etherealengine/server-core/src/hooks/set-loggedin-user-in-query`?

Reply:
it's not needed, I replaced it with `set-loggedin-user-in-query`